toolchain/check/diagnostic_helpers.h

```diff
@@ -10,11 +10,7 @@
 #include "toolchain/sem_ir/ids.h"
 
 namespace Carbon::Check {
-
-// TODO: Consider instead changing calls to `SemIR::LocId::TokenOnly(...)`.
-inline auto TokenOnly(SemIR::LocId loc_id) -> SemIR::LocId {
-  return loc_id.ToTokenOnly();
-}
+class Context;
```

Comment:
Noticed while glancing again at the PR, looks leftover

```suggestion
```

Reply:
Done